cves/CVE-2015-2317.yml

```diff
@@ -144,10 +134,10 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: "Looked at commit messages around fix and on project bug tracker"
```

Comment:
Have you tried looking at the security release? That might have some information.

Reply:
Yes, there was no info there